### ISSUE TITLE
Error after installing

### ISSUE THREAD

**Issue by angelod1as** [kavod-io/vscode-jest-test-adapter#112]
**Describe the bug**
- [ ] test load missing suites/tests
- [x] test load errors
- [ ] test running errors

The extension should work but returns an error.

**To Reproduce**
1. Install the extension
2. Get error

**Expected behavior**
The extension should work properly (I've never been able to see it working, so I don't know what to expect)

**Error Log**
```
'Usage: test.js [--config=<pathToConfigFile>] [TestPathPattern]\n' +
  '\n' +
  'Opções:\n' +
  '  --help, -h                    Exibe ajuda                           [booleano]\n' +
  '  --version, -v                 Print the version and exit            [booleano]\n' +
  '  --all                         The opposite of `onlyChanged`. If `onlyChanged`\n' +
  '                                is set by default, running jest with `--all`\n' +
  '                                will force Jest to run all tests instead of\n' +
  '                                running only tests related to changed files.\n' +
  '                                                                      [booleano]\n' +
  '  --automock                    Automock all files by default.        [booleano]\n' +
  '  --bail, -b                    Exit the test suite immediately after `n` number\n' +
  '                                of failing tests.                     [booleano]\n' +
  '  --browser                     Respect the "browser" field in package.json when\n' +
  '                                resolving modules. Some packages export\n' +
  '                                different versions based on whether they are\n' +
  '                                operating in node.js or a browser.    [booleano]\n' +
  '  --cache                       Whether to use the transform cache. Disable the\n' +
  '                                cache using --no-cache.               [booleano]\n' +
  '  --cacheDirectory              The directory where Jest should store its cached\n' +
  '                                dependency information.                 [string]\n' +
  '  --changedFilesWithAncestor    Runs tests related to the current changes and\n' +
  '                                the changes made in the last commit. Behaves\n' +
  '                                similarly to `--onlyChanged`.         [booleano]\n' +
  '  --changedSince                Runs tests related to the changes since the\n' +
  '                                provided branch. If the current branch has\n' +
  '                                diverged from the given branch, then only\n' +
  '                                changes made locally will be tested. Behaves\n' +
  '                                similarly to `--onlyChanged`.           [string]\n' +
  '  --ci                          Whether to run Jest in continuous integration\n' +
  '                                (CI) mode. This option is on by default in most\n' +
  '                                popular CI environments. It will  prevent\n' +
  '                                snapshots from being written unless explicitly\n' +
  '                                requested.             [booleano] [padrão: true]\n' +
  '  --clearCache                  Clears the configured Jest cache directory and\n' +
  '                                then exits. Default directory can be found by\n' +
  '                                calling jest --showConfig             [booleano]\n' +
  '  --clearMocks                  Automatically clear mock calls and instances\n' +
  '                                between every test. Equivalent to calling\n' +
  '                                jest.clearAllMocks() between each test.\n' +
  '                                                                      [booleano]\n' +
  '  --collectCoverage             Alias for --coverage.                 [booleano]\n' +
  '  --collectCoverageFrom         A glob pattern relative to <rootDir> matching\n' +
  '                                the files that coverage info needs to be\n' +
  '                                collected from.                         [string]\n' +
  '  --collectCoverageOnlyFrom     Explicit list of paths coverage will be\n' +
  '                                restricted to.                           [array]\n' +
  '  --color                       Forces test results output color highlighting\n' +
  '                                (even if stdout is not a TTY). Set to false if\n' +
  '                                you would like to have no colors.     [booleano]\n' +
  '  --colors                      Alias for `--color`.                  [booleano]\n' +
  '  --config, -c                  The path to a jest config file specifying how to\n' +
  '                                find and execute tests. If no rootDir is set in\n' +
  '                                the config, the directory containing the config\n' +
  '                                file is assumed to be the rootDir for the\n' +
  '                                project.This can also be a JSON encoded value\n' +
  '                                which Jest will use as configuration.   [string]\n' +
  '  --coverage                    Indicates that test coverage information should\n' +
  '                                be collected and reported in the output.\n' +
  '                                                                      [booleano]\n' +
  '  --coverageDirectory           The directory where Jest should output its\n' +
  '                                coverage files.                         [string]\n' +
  '  --coveragePathIgnorePatterns  An array of regexp pattern strings that are\n' +
  '                                matched against all file paths before executing\n' +
  '                                the test. If the file pathmatches any of the\n' +
  '                                patterns, coverage information will be skipped.\n' +
  '                                                                         [array]\n' +
  '  --coverageReporters           A list of reporter names that Jest uses when\n' +
  '                                writing coverage reports. Any istanbul reporter\n' +
  '                                can be used.                             [array]\n' +
  '  --coverageThreshold           A JSON string with which will be used to\n' +
  '                                configure minimum threshold enforcement for\n' +
  '                                coverage results                        [string]\n' +
  '  --debug                       Print debugging info about your jest config.\n' +
  '                                                                      [booleano]\n' +
  '  --detectLeaks                 **EXPERIMENTAL**: Detect memory leaks in tests.\n' +
  '                                After executing a test, it will try to garbage\n' +
  '                                collect the global object used, and fail if it\n' +
  '                                was leaked            [booleano] [padrão: false]\n' +
  '  --detectOpenHandles           Print out remaining open handles preventing Jest\n' +
  '                                from exiting at the end of a test run. Implies\n' +
  '                                `runInBand`.          [booleano] [padrão: false]\n' +
  '  --env                         The test environment used for all tests. This\n' +
  '                                can point to any file or node module. Examples:\n' +
  '                                `jsdom`, `node` or `path/to/my-environment.js`\n' +
  '                                                                        [string]\n' +
  '  --errorOnDeprecated           Make calling deprecated APIs throw helpful error\n' +
  '                                messages.             [booleano] [padrão: false]\n' +
  '  --expand, -e                  Use this flag to show full diffs instead of a\n' +
  '                                patch.                                [booleano]\n' +
  '  --filter                      Path to a module exporting a filtering function.\n' +
  '                                This method receives a list of tests which can\n' +
  '                                be manipulated to exclude tests from running.\n' +
  '                                Especially useful when used in conjunction with\n' +
  '                                a testing infrastructure to filter known broken\n' +
  '                                tests.                                  [string]\n' +
  '  --findRelatedTests            Find related tests for a list of source files\n' +
  '                                that were passed in as arguments. Useful for\n' +
  '                                pre-commit hook integration to run the minimal\n' +
  '                                amount of tests necessary.            [booleano]\n' +
  '  --forceExit                   Force Jest to exit after all tests have\n' +
  '                                completed running. This is useful when resources\n' +
  '                                set up by test code cannot be adequately cleaned\n' +
  '                                up.                                   [booleano]\n' +
  '  --globalSetup                 The path to a module that runs before All Tests.\n' +
  '                                                                        [string]\n' +
  '  --globalTeardown              The path to a module that runs after All Tests.\n' +
  '                                                                        [string]\n' +
  '  --globals                     A JSON string with map of global variables that\n' +
  '                                need to be available in all test environments.\n' +
  '                                                                        [string]\n' +
  '  --haste                       A JSON string with map of variables for the\n' +
  '                                haste module system                     [string]\n' +
  '  --init                        Generate a basic configuration file   [booleano]\n' +
  '  --json                        Prints the test results in JSON. This mode will\n' +
  '                                send all other test output and user messages to\n' +
  '                                stderr.                               [booleano]\n' +
  '  --lastCommit                  Run all tests affected by file changes in the\n' +
  '                                last commit made. Behaves similarly to\n' +
  '                                `--onlyChanged`.                      [booleano]\n' +
  '  --listTests                   Lists all tests Jest will run given the\n' +
  '                                arguments and exits. Most useful in a CI system\n' +
  '                                together with `--findRelatedTests` to determine\n' +
  '                                the tests Jest will run based on specific files\n' +
  '                                                      [booleano] [padrão: false]\n' +
  '  --logHeapUsage                Logs the heap usage after every test. Useful to\n' +
  '                                debug memory leaks. Use together with\n' +
  '                                `--runInBand` and `--expose-gc` in node.\n' +
  '                                                                      [booleano]\n' +
  '  --mapCoverage                 Maps code coverage reports against original\n' +
  '                                source code when transformers supply source\n' +
  '                                maps.\n' +
  '\n' +
  '                                DEPRECATED                            [booleano]\n' +
  '  --maxConcurrency              Specifies the maximum number of tests that are\n' +
  '                                allowed to runconcurrently. This only affects\n' +
  '                                tests using `test.concurrent`.\n' +
  '                                                            [número] [padrão: 5]\n' +
  '  --maxWorkers, -w              Specifies the maximum number of workers the\n' +
  '                                worker-pool will spawn for running tests. This\n' +
  '                                defaults to the number of the cores available on\n' +
  '                                your machine. (its usually best not to override\n' +
  '                                this default)                           [string]\n' +
  '  --moduleDirectories           An array of directory names to be searched\n' +
  "                                recursively up from the requiring module's\n" +
  '                                location.                                [array]\n' +
  '  --moduleFileExtensions        An array of file extensions your modules use.\n' +
  '                                If you require modules without specifying a\n' +
  '                                file extension, these are the extensions Jest\n' +
  '                                will look for.                           [array]\n' +
  '  --moduleNameMapper            A JSON string with a map from regular\n' +
  '                                expressions to module names that allow to stub\n' +
  '                                out resources, like images or styles with a\n' +
  '                                single module                           [string]\n' +
  '  --modulePathIgnorePatterns    An array of regexp pattern strings that are\n' +
  '                                matched against all module paths before those\n' +
  '                                paths are to be considered "visible" to the\n' +
  '                                module loader.                           [array]\n' +
  '  --modulePaths                 An alternative API to setting the NODE_PATH env\n' +
  '                                variable, modulePaths is an array of absolute\n' +
  '                                paths to additional locations to search when\n' +
  '                                resolving modules.                       [array]\n' +
  '  --noStackTrace                Disables stack trace in test results output\n' +
  '                                                                      [booleano]\n' +
  '  --notify                      Activates notifications for test results.\n' +
  '                                                                      [booleano]\n' +
  '  --notifyMode                  Specifies when notifications will appear for\n' +
  '                                test results.[string] [padrão: "failure-change"]\n' +
  '  --onlyChanged, -o             Attempts to identify which tests to run based on\n' +
  '                                which files have changed in the current\n' +
  "                                repository. Only works if you're running tests\n" +
  '                                in a git or hg repository at the moment.\n' +
  '                                                                      [booleano]\n' +
  '  --onlyFailures, -f            Run tests that failed in the previous execution.\n' +
  '                                                                      [booleano]\n' +
  '  --outputFile                  Write test results to a file when the --json\n' +
  '                                option is also specified.               [string]\n' +
  '  --passWithNoTests             Will not fail if no tests are found (for example\n' +
  '                                while using `--testPathPattern`.)\n' +
  '                                                      [booleano] [padrão: false]\n' +
  "  --preset                      A preset that is used as a base for Jest's\n" +
  '                                configuration.                          [string]\n' +
  '  --prettierPath                The path to the "prettier" module used for\n' +
  '                                inline snapshots.                       [string]\n' +
  '  --projects                    A list of projects that use Jest to run all\n' +
  '                                tests of all projects in a single instance of\n' +
  '                                Jest.                                    [array]\n' +
  '  --reporters                   A list of custom reporters for the test suite.\n' +
  '                                                                         [array]\n' +
  '  --resetMocks                  Automatically reset mock state between every\n' +
  '                                test. Equivalent to calling jest.resetAllMocks()\n' +
  '                                between each test.                    [booleano]\n' +
  '  --resetModules                If enabled, the module registry for every test\n' +
  '                                file will be reset before running each\n' +
  '                                individual test.                      [booleano]\n' +
  '  --resolver                    A JSON string which allows the use of a custom\n' +
  '                                resolver.                               [string]\n' +
  '  --restoreMocks                Automatically restore mock state and\n' +
  '                                implementation between every test. Equivalent to\n' +
  '                                calling jest.restoreAllMocks() between each\n' +
  '                                test.                                 [booleano]\n' +
  '  --rootDir                     The root directory that Jest should scan for\n' +
  '                                tests and modules within.               [string]\n' +
  '  --roots                       A list of paths to directories that Jest should\n' +
  '                                use to search for files in.              [array]\n' +
  '  --runInBand, -i               Run all tests serially in the current process\n' +
  '                                (rather than creating a worker pool of child\n' +
  '                                processes that run tests). This is sometimes\n' +
  '                                useful for debugging, but such use cases are\n' +
  '                                pretty rare.                          [booleano]\n' +
  '  --runTestsByPath              Used when provided patterns are exact file\n' +
  '                                paths. This avoids converting them into a\n' +
  '                                regular expression and matching it against every\n' +
  '                                single file.          [booleano] [padrão: false]\n' +
  "  --runner                      Allows to use a custom runner instead of Jest's\n" +
  '                                default test runner.                    [string]\n' +
  '  --setupFiles                  A list of paths to modules that run some code\n' +
  '                                to configure or set up the testing environment\n' +
  '                                before each test.                        [array]\n' +
  '  --setupFilesAfterEnv          A list of paths to modules that run some code\n' +
  '                                to configure or set up the testing framework\n' +
  '                                before each test                         [array]\n' +
  '  --showConfig                  Print your jest config and then exits.[booleano]\n' +
  '  --silent                      Prevent tests from printing messages through the\n' +
  '                                console.                              [booleano]\n' +
  '  --skipFilter                  Disables the filter provided by --filter. Useful\n' +
  '                                for CI jobs, or local enforcement when fixing\n' +
  '                                tests.                                [booleano]\n' +
  '  --snapshotSerializers         A list of paths to snapshot serializer modules\n' +
  '                                Jest should use for snapshot testing.    [array]\n' +
  '  --testEnvironment             Alias for --env                         [string]\n' +
  '  --testEnvironmentOptions      Test environment options that will be passed to\n' +
  '                                the testEnvironment. The relevant options depend\n' +
  '                                on the environment.                     [string]\n' +
  '  --testFailureExitCode         Exit code of `jest` command if the test run\n' +
  '                                failed                                  [string]\n' +
  '  --testLocationInResults       Add `location` information to the test results\n' +
  '                                                      [booleano] [padrão: false]\n' +
  '  --testMatch                   The glob patterns Jest uses to detect test\n' +
  '                                files.                                   [array]\n' +
  '  --testNamePattern, -t         Run only tests with a name that matches the\n' +
  '                                regex pattern.                          [string]\n' +
  '  --testPathIgnorePatterns      An array of regexp pattern strings that are\n' +
  '                                matched against all test paths before executing\n' +
  '                                the test. If the test path matches any of the\n' +
  '                                patterns, it will be skipped.            [array]\n' +
  '  --testPathPattern             A regexp pattern string that is matched against\n' +
  '                                all tests paths before executing the test.\n' +
  '                                                                         [array]\n' +
  '  --testRegex                   A string or array of string regexp patterns that\n' +
  '                                Jest uses to detect test files.          [array]\n' +
  '  --testResultsProcessor        Allows the use of a custom results processor.\n' +
  '                                This processor must be a node module that\n' +
  '                                exports a function expecting as the first\n' +
  '                                argument the result object.             [string]\n' +
  '  --testRunner                  Allows to specify a custom test runner. The\n' +
  '                                default is  `jasmine2`. A path to a custom test\n' +
  '                                runner can be provided:\n' +
  '                                `<rootDir>/path/to/testRunner.js`.      [string]\n' +
  '  --testSequencer               Allows to specify a custom test sequencer. The\n' +
  '                                default is `@jest/test-sequencer`. A path to a\n' +
  '                                custom test sequencer can be provided:\n' +
  '                                `<rootDir>/path/to/testSequencer.js`    [string]\n' +
  '  --testTimeout                 This option sets the default timeouts of test\n' +
  '                                cases.                                  [número]\n' +
  '  --testURL                     This option sets the URL for the jsdom\n' +
  '                                environment.                            [string]\n' +
  '  --timers                      Setting this value to fake allows the use of\n' +
  '                                fake timers for functions such as setTimeout.\n' +
  '                                                                        [string]\n' +
  '  --transform                   A JSON string which maps from regular\n' +
  '                                expressions to paths to transformers.   [string]\n' +
  '  --transformIgnorePatterns     An array of regexp pattern strings that are\n' +
  '                                matched against all source file paths before\n' +
  '                                transformation.                          [array]\n' +
  '  --unmockedModulePathPatterns  An array of regexp pattern strings that are\n' +
  '                                matched against all modules before the module\n' +
  '                                loader will automatically return a mock for\n' +
  '                                them.                                    [array]\n' +
  '  --updateSnapshot, -u          Use this flag to re-record snapshots. Can be\n' +
  '                                used together with a test suite pattern or with\n' +
  '                                `--testNamePattern` to re-record snapshot for\n' +
  '                                test matching the pattern             [booleano]\n' +
  '  --useStderr                   Divert all output to stderr.          [booleano]\n' +
  '  --verbose                     Display individual test results with the test\n' +
  '                                suite hierarchy.                      [booleano]\n' +
  '  --watch                       Watch files for changes and rerun tests related\n' +
  '                                to changed files. If you want to re-run all\n' +
  '                                tests when a file has changed, use the\n' +
  '                                `--watchAll` option.                  [booleano]\n' +
  '  --watchAll                    Watch files for changes and rerun all tests. If\n' +
  '                                you want to re-run only the tests related to the\n' +
  '                                changed files, use the `--watch` option.\n' +
  '                                                                      [booleano]\n' +
  '  --watchPathIgnorePatterns     An array of regexp pattern strings that are\n' +
  '                                matched against all paths before trigger test\n' +
  '                                re-run in watch mode. If the test path matches\n' +
  '                                any of the patterns, it will be skipped. [array]\n' +
  '  --watchman                    Whether to use watchman for file crawling.\n' +
  '                                Disable using --no-watchman.          [booleano]\n' +
  '\n' +
  'Documentation: https://jestjs.io/\n' +
  '\n' +
  'argv.config.match is not a function\n' +
  'npm ERR! code ELIFECYCLE\n' +
  'npm ERR! errno 1\n' +
  'npm ERR! performance-web@0.1.1 test: `react-app-rewired test --env=jsdom --no-cache "--projects" "/home/angelo/Documentos/GIT/professional/seasoned/metadados-performance-web/jest.config.js" "--showConfig" "--config" "/home/angelo/Documentos/GIT/professional/seasoned/metadados-performance-web/jest.config.js"`\n' +
  'npm ERR! Exit status 1\n' +
  'npm ERR! \n' +
  'npm ERR! Failed at the performance-web@0.1.1 test script.\n' +
  'npm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n' +
  '\n' +
  'npm ERR! A complete log of this run can be found in:\n' +
  'npm ERR!     /home/angelo/.npm/_logs/2021-02-26T13_58_55_508Z-debug.log\n'
```

**Screenshots**
![image](https://user-images.githubusercontent.com/13950513/109309623-4a8e6800-7822-11eb-82ca-10da05978299.png)

**Link to source**
Unfortunately can't link source, but will tell detailed information if asked.

**Desktop (please complete the following information):**
 - OS: Pop!_OS

**Additional context**
Jest works with no issues, both in terminal and using [Jest](https://marketplace.visualstudio.com/items?itemName=Orta.vscode-jest) extension

**Comment by rossknudsen** [comment]
So it looks like you're using CRA rewired. Which is fine, but it probably matches my CRA handling code, which may not be compatible with that code.

**Comment by angelod1as** [comment]
I'm truly sorry, but I didn't understand what you said. I don't know what CRA stands for. So, does that mean it's not solveable?

**Comment by rossknudsen** [comment]
CRA = Create React App. The rewired variant allows you to tweak some stuff without having to eject.

In terms of solving, its a matter of adding custom code to handle that scenario.

**Comment by rossknudsen** [comment]
Can you post jest.config.js also?

**Comment by angelod1as** [comment]
This is my jest.config.js

```
module.exports = {
  snapshotSerializers: ['jest-emotion'],
}
```

**Comment by JustAnOldG** [comment]
I was able to solve this problem for me when I was reading through issue https://github.com/kavod-io/vscode-jest-test-adapter/issues/113.
I had Jest configuration in my package.json and after removing it and restarting VSCode, I was able to see my tests listed in the test explorer again.

Just pointing this out however it is a temp solution for me as I am using Create React App (CRA) and that is how you setup Jest config. I am not sure if CRA will read a jest config file.
